packages/common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uma/common",
-  "version": "2.0.1",
+  "version": "2.0.3",
```

Comment:
nit: do we need to release a new version? If you install `2.0.1` fresh, it picks up the patch automatically since truffle-ledger-provider has a `^`. Also why a 2-version bump?

Reply:
that works for me and makes sence! Let's get the #2689 PR merged and then we can rebase this onto of that one with the 2.0.2 version.